main.py

```diff
@@ -797,7 +797,8 @@ async def perform_filters(request: PerformFiltersRequest):
         
         # Map of filter names to their pipeline IDs
         filter_map = {
-            "NSFW Filter": "nsfw_filter_pipeline"
+            "NSFW Filter": "nsfw_filter_pipeline",
+            "Jailbreak Filter": "jailbreak_filter_pipeline"
         }
```

Comment:
what is this change for? i remember adding a hallucination filter without a change in main.py

Reply:
This is to map the pipeline based on the filters given by the api key